_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -148,14 +181,31 @@ Label is included in accessibility tree, but not visible, and does not describe
 <input aria-labelledby="label_fname" type="text" name="fname"/>
 ```
 
+#### Failed example 6
+
+Accessible name created through "aria-label" does not describes the purpose of the associated element.
+
+```html
+<input aria-label="Menu" id="fname" type="text" name="fname"/>
+```
+
+#### Failed example 7
+
+The `label` in itself does not describe the purpose of the form field, and while the context can be relied upon to differentiate form fields from each other, it is not sufficient to rely on the context for describing the entire purpose of the form field.
+
+```html
+<h2>Name</h2>
+<label>Fill in: <input id="name" type="text" name="name"/></label>
+````
+
```

Comment:
Can we add an example illustrating the accessible name computation, in particular that the aria-label relationship completly delete the label for relation :

<label for="fname">Nom</label>
<input aria-label="Menu" id="fname" type="text" name="fname"/>

Reply:
Added a Passed example 8 + Failed example 8 for this now :)